js/pulse.js

```diff
@@ -204,27 +204,27 @@ var starred = {
 
 /* Detailer View Handler */
 
```

Comment:
changes to this file:

- Renamed `enableDetailView()` to `DetailViewManager` and make it an `object` instead
- Added `init()` method to it
- Pretty much just cloning a project's thumbnail HTML and put it onto the `detail-view` wrapper.
- Made sure `shareBtnClickHandler` is attached to all `share-btn` (thumbnail ones and the one on detail view)